CROSSSECTOR/IT/Tests/schema.json

```diff
@@ -0,0 +1,72 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "$schemaVersion": "0.0.1",
+  "$id": "https://smart-data-models.github.io/Testing/Tests/schema.json",
+  "title": "Smart Data models - Testing schema",
+  "modelTags": "",
+  "description": "This is the data model for representing the result of testing procedures.",
+  "type": "object",
+  "required": [
+    "id",
+    "type"
+  ],
+  "allOf": [
+    {
+      "properties": {
+        "timestamp": {
+          "type": "number",
+          "minimum": 0,
+          "multipleOf": 1,
+          "description": "Property. Model:'https://schema.org/Number'. EPOCH timestamp measures in milliseconds corresponding to the starting point of the execution of the tests."
+        },
+        "elapsedTime": {
+          "type": "number",
+          "minimum": 0,
+          "multipleOf": 1,
+          "description": "Property. Model:'https://schema.org/Number'. Elapsed time since the beginning of the execution of the tests. It measures in milliseconds the duration of the execution of the tests."
+        },
+        "status": {
+          "type": "string",
+          "description": "Property. Model:'https://schema.org/Text'. Overall status of the execution of the tests: OK, all the tests passed; NOK, there is at least some tests that failed; POK, it is possible that some tests are marked as optionals, therefore it they fails the overall status is not NOK but POK.",
+          "enum": [
```

Comment:
I guess that this schema does not validate some of the payloads because you have N/A, etc

Reply:
I do not understand this point. Nevertheless, to prevent any error the concept Not Applicable (N/A), I will change to NA

Reply:
DISMISS?